src/pages/Transaction/TransactionCellScript/CellInfoDataView.tsx

```diff
@@ -0,0 +1,74 @@
+import { useMemo, useRef, useState } from 'react'
+import { Select } from 'antd'
+import { useTranslation } from 'react-i18next'
+import { Root, Trigger, Portal, Content } from 'selection-popover'
+import styles from './styles.module.scss'
+
+const DECODER = {
+  utf8: 'UTF-8',
+  number: 'Number',
+  // todo: support decode address & Big-Endian
+  // bigend: 'Big-Endian',
+}
+
+export const CellInfoDataView: React.FC<{ data: string }> = ({ data }) => {
+  const { t } = useTranslation()
+  const ref = useRef(null)
+  const wrapperRef = useRef(null)
+  const [selectedText, setSelectedText] = useState('')
+  const [decoder, setDecoder] = useState<keyof typeof DECODER>(Object.keys(DECODER)[0] as keyof typeof DECODER)
+
+  const setSelection = () => {
+    const selection = window.getSelection()
+    const text = (selection?.anchorNode?.textContent ?? '').slice(selection?.anchorOffset, selection?.focusOffset)
+
+    setSelectedText(text)
+  }
+
+  const decodedText = useMemo(() => {
+    if (decoder === 'utf8') {
+      const bytes = [...selectedText.matchAll(/[0-9a-f]{2}/g)].map(a => parseInt(a[0], 16))
+      return new TextDecoder().decode(new Uint8Array(bytes))
+    }
+
+    if (decoder === 'number') {
+      return parseInt(selectedText, 16).toString()
```

Comment:
Use `BigNumber`, otherwise some values will be displayed in scientific notation.